src/crosschain/swapExactIn/changellySwap/changellySwap.ts

```
@@ -1,20 +1,33 @@
 import { AddressZero } from '@ethersproject/constants'
 
+import { ChainId } from '../../../constants'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(`@typescript-eslint/consistent-type-imports`)

---

[error] Replace `⏎····isSolanaChainId,⏎····isTonChainId,⏎····isTronChainId,⏎····type·TronTransactionData,⏎` with `·isSolanaChainId,·isTonChainId,·isTronChainId,·type·TronTransactionData·`

(prettier/prettier)